docs/source/WebAPI.rst

```diff
@@ -1,6 +1,8 @@
 Web API
 =======
 
+For Swagger page See `<openapi.html>`_ or `<swagger.html>`_
```

Comment:
Don't we have to define the target for the links? Also, instead of swagger, maybe we can use something like structure of the api here, to be more beginner friendly.

Reply:
This is just a reference link to the swagger pages, what is the issue here?